api/src/main/java/com/datastrato/gravitino/authorization/SecurableObjects.java

```diff
@@ -38,27 +76,39 @@ public static SecurableObject of(String... names) {
       if (name.equals("*")) {
         throw new IllegalArgumentException(
             "Cannot create a securable object with `*` name. If you want to use a securable object which represents all catalogs,"
```

Comment:
all metalakes, not all catalogs.

Reply:
I will modify the comment. I will explain ofMetalake and ofAllMetalakes.